app/helpers/checklist_tests_helper.rb

```diff
@@ -59,6 +59,19 @@ def coded_attribute?(criteria)
   def lookup_valueset_name(oid)
     vs = HealthDataStandards::SVS::ValueSet.where(oid: oid)
     return oid unless vs && vs.first
-    "#{vs.first.display_name}: #{oid}"
+    "#{vs.first.display_name}"
+  end
+
+
+  def lookup_valueset_oid(oid)
+    vs = HealthDataStandards::SVS::ValueSet.where(oid: oid)
+    return oid unless vs && vs.first
+    "#{oid}"
```

Comment:
But this function takes `oid` and always just returns `oid`. So it doesn't do anything. If this is in fact the behavior you want, you can just use `oid` directly wherever this function is called.